mass_editing/tests/test_mass_editing.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module uses OpenERP, Open Source Management Solution Framework.
+#    Copyright (C):
+#        2012-Today Serpent Consulting Services (<http://www.serpentcs.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>
+#
+##############################################################################
+
+from openerp.tests import common
+
+
+class TestMassEditing(common.TransactionCase):
+
+    def setUp(self):
+        super(TestMassEditing, self).setUp()
+        self.mass_object_model = self.env['mass.object']
+        self.res_partner_model = self.env['res.partner']
+        self.partner = self._create_partner()
+        self.partner_model = self.env['ir.model'].\
```

Comment:
IIRC `self.env['res.partner']._model` would also work, and would skip the DB search